ipapython/ntpmethods.py

```diff
@@ -0,0 +1,215 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+from __future__ import absolute_import
+
+import shutil
+import re
+import sys
+
+# pylint: disable=import-error,ipa-forbidden-import
+from ipaclient.install import ipadiscovery  # pylint: disable=E0401,E0611
+# pylint: enable=import-error,ipa-forbidden-import
+from ipaplatform import services
+from ipaplatform.paths import paths
+from ipaplatform.constants import constants
+from ipapython import ipautil
+
+
+def service_command():
+    timedata_srv = {
+        'openntpd': {
+            'api': services.knownservices.openntpd,
+            'service': 'openntpd',
+        },
+        'ntpd': {
+            'api': services.knownservices.ntpd,
+            'service': 'ntpd',
+        },
+        'chrony': {
+            'api': services.knownservices.chronyd,
+            'service': 'chronyd'
+        }
+    }
+
+    return timedata_srv[detect_time_server()]
```

Comment:
Because of this deprecation runtime warning:
> And this results into: 
> ```
> /usr/lib/python3.7/site-packages/ipaplatform/base/services.py:207: RuntimeWarning: RedHatService('chronyd', > api=None) is deprecated.
>   super(SystemdService, self).__init__(service_name, api=api)
> WARNING: conflicting time&date synchronization service 'chronyd' will be disabled
> in favor of chrony
> ```
We should reconsider using `['api']` as you use it:
```python
service_command()['api'].is_enabled()
```
and maybe return objects like:
```
services.knownservices.chronyd
services.knownservices.ntpd
services.knownservices.ntpd
```
so in the end we will use for example:
`service_command().stop()`
and handle service name otherwise.

@mrandybu  please let us discuss this.